apps/mainsite/account_adapter.py

```diff
@@ -49,6 +177,15 @@ def send_mail(self, template_prefix, email, context):
         self.EMAIL_FROM_STRING = self.set_email_string(context)
 
         msg = self.render_mail(template_prefix, email, context)
+        # badge_id is equal to the badge instance slug
+        pdf_document = generate_pdf_content(context['badge_id'])
```

Comment:
I did approve the PR then I found an error when creating a new account using fronted. 
<img width="569" alt="Screenshot 2024-05-22 at 23 18 26" src="https://github.com/mint-o-badges/badgr-server/assets/40634685/0e53a57b-9fc8-4ec6-8cde-f3095cb93bee">